doc/src/cylc-user-guide/cug.tex

```diff
@@ -1220,53 +1272,15 @@ \subsection{Viewing Suite Logs via Web Browser: Cylc Review}
 \label{fig-review-screenshot}
 \end{figure}
 
-If a Cylc Review server is provided at your site, you can open the Cylc
-Review page for a suite by running the \lstinline=cylc review= command.
-See~\ref{HostsforCylcReview} for requirements and~\ref{ConfiguringCylcReview}
-for configuration steps for setting up a host to run the service at your site.
-
-Otherwise an ad-hoc web server can be set up using the
-\lstinline=cylc review start= command argument.
+Otherwise, to start an ad hoc Cylc Review service to view your own suite logs
+(or those of others, if you have read access to them), run:
 
-\subsubsection{Hosts For Running Cylc Review}
-\label{HostsforCylcReview}
+\lstinline=setsid cylc review start 0</dev/null 1>/dev/null 2>&1 &=
 
-Connectivity requirements:
-
-\begin{myitemize}
-
-\item Must be able to access the home directories of users' Cylc run
-directories.
-
-\end {myitemize}
-
-\subsubsection{Configuring Cylc Review}
-\label{ConfiguringCylcReview}
-
-Cylc Review can provide an intranet web service at your site for users to
-view their suite logs using a web browser. Depending on settings at your
-site, you may or may not be able to set up this service
-(see~\ref{HostsforCylcReview}).
-
-You can start an ad-hoc Cylc Review web server by running:
-
-\hilight{\lstinline=setsid /path/to/../cylc review start 0</dev/null 1>/dev/null 2>\&1 \&=}
-
-You will find the access and error logs under \lstinline=~/.cylc/cylc-review*=.
-
-Alternatively you can run the Cylc Review web service under Apache
-\lstinline=mod_wsgi=. To do this you will need to set up an Apache module
-configuration file (typically in
-\lstinline=/etc/httpd/conf.d/cylc-wsgi.conf=) containing the following (with
-the paths set appropriately):
-
-\lstset{language=bash}
-\begin{lstlisting}
-WSGIPythonPath /path/to/cylc/lib
-WSGIScriptAlias /cylc-review /path/to/lib/cylc/review.py
-\end{lstlisting}
-
-Use the Apache log at e.g. \lstinline=/var/log/httpd/= to debug problems.
+The service should start at \lstinline=http://<server>:8080= (the port number
+can optionally be set on the command line). Service logs are written to
+\lstinline=~/.cylc/cylc-review*=. Run \lstinline=cylc review status= to view
```

Comment:
Strictly the status command should simply be ``cylc review``, not ``cylc review status``. This is as documented under the docstring & output:

```bash
$ cylc review --help                 
Usage: cylc [info] review [OPTIONS] [start [PORT]] [stop] 
...
With no arguments, the status of the ad-hoc web service server is printed.
...
```
I had a play around & ``cylc review status`` does actually produce the same result, but only because any argument (or precisely any number not ``too many`` i.e. ``<=2``) other than the valid ``stop`` or ``start`` will not be recognised so that the logic applied for no arguments is run. E.g.:

```bash
$ cylc review any nonsense
host=0.0.0.0
pid=7395
port=8080
```

Reply:
For context, this originates from Rose Bush. Running after starting it as an ad-hoc server:

```bash
$ rose bush anything really
host=0.0.0.0
pid=13839
port=4100

```